test/fixtures/shared/terraform.tfvars

```diff
@@ -0,0 +1,2 @@
+project_id = ""
```

Comment:
There is no valid default for `project_id`.

Reply:
I have simply renamed file